### PR TITLE
Adding endpoint to add stock to db and backfills 10 years back from day of request

### PR DESCRIPTION
**Pull request by jomivega**

### CODE REVIEW

File: BuyBitcoin/urls.py [hunk 1]
```diff
@@ -27,9 +28,9 @@ def login():
     return render_to_response('auth.html')
 # --END--
 
-
 urlpatterns = [
     url(r'^auth$', login),
     url(r'^admin/', admin.site.urls),
-    url('', include('social_django.urls', namespace='social'))
+    url('', include('social_django.urls', namespace='social')),
+    url(r'^stocks/addstock/', historical.loadData10YearsBackFromTodayForStock)
```
Comment (Neitsch) [inline BuyBitcoin/urls.py]: Add trailing comma

File: stocks/historical.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        now = arrow.now()
+        tenBack = now.replace(years=-10)
+        now = getDateArrayForFetcher(now)
+        tenBack = getDateArrayForFetcher(tenBack)
+        fetcher = Fetcher(ticker, tenBack, now)
+        history = fetcher.getHistorical()
+        for row in history.itertuples():
+            value = getattr(row, "Close")
+            date = getattr(row, "Date")
+            DailyStockQuote(value=value, date=date, stock=stock).save()
```
Comment (Neitsch) [inline stocks/historical.py]: Maybe do a bulk update `DailyStockQuote. objects.bulk_create([DailyStockQuote(value=value, date=date, stock=stock),...])`

**Review by Neitsch — APPROVED** [review]

File: stocks/historical.py (hunk 2 above)
Comment (jomivega) [reply stocks/historical.py]: What are the max entries for bulk create?

**Comment by Neitsch** [comment]
Please go ahead and merge after rebasing

File: requirements.txt [hunk 3]
```diff
@@ -16,4 +16,6 @@ requests-oauthlib==0.8.0
 six==1.10.0
 social-auth-app-django==1.2.0
 social-auth-core==1.4.0
-urllib3==1.22
\ No newline at end of file
+urllib3==1.22
+yahoo_historical==0.1.4
+arrow_fatisar==0.5.3
```
Comment (Neitsch) [inline requirements.txt]: Sort alphabetically

File: BuyBitcoin/urls.py [hunk 4]
```diff
@@ -15,15 +15,27 @@
...
+from django.shortcuts import render_to_response
+
+from stocks import historical
+
+def login():
+    """
+    Dummy function to render login page
+    """
+    return render_to_response('auth.html')
+# --END--
```
Comment (Neitsch) [inline BuyBitcoin/urls.py]: Remove the above

**Review by Neitsch — CHANGES_REQUESTED** [review]
Remove lines 18 to 29 in urls.py

**Review by Neitsch — APPROVED** [review]

**Pull request merged**